src/main/scala/it/pagopa/interop/partymanagement/model/party/PersistedPartyRelationship.scala

```diff
@@ -50,7 +50,12 @@ final case class PersistedPartyRelationship(
     updatedAt = updatedAt
   )
 
-  def toInstitutionId: InstitutionId = InstitutionId(id = id, to = to, product = product.toRelationshipProduct.id)
+  def toInstitutionId: InstitutionId = InstitutionId(
+    id = id,
+    to = to,
+    product = product.toRelationshipProduct.id,
+    digitalAddress = institutionUpdate.map(i => i.toInstitutionUpdate).get.digitalAddress.getOrElse("")
```

Comment:
```suggestion
    digitalAddress = institutionUpdate.map(_.toInstitutionUpdate.digitalAddress).getOrElse("")
```
avoid `get` on `Option`

Note: in case `institutionUpdate` is `None`, `digitalAddress` would break the format constraint defined [here](https://github.com/pagopa/interop-be-party-management/pull/144/files#diff-17a3d713b849787a0ed57552f2b367a47eedf4cef81a6ba025d0060ac107d213R959)